packages/server/src/main/resources/application.properties

```diff
@@ -32,8 +32,13 @@ terarium.clientConfig.clientLogShippingIntervalMillis=3000
 ########################################################################################################################
 terarium.keycloak.url=https://keycloak.staging.terarium.ai
 spring.security.oauth2.client.provider.keycloak.issuer-uri=${terarium.keycloak.url:localhost}/realms/${terarium.keycloak.realm:Terarium}
+#spring.security.oauth2.client.provider.keycloak.user-name-attribute=preferred_username
```

Comment:
If none of these are used, should we leave out?

Reply:
great question, I had commented them out because I worried they would be needed - and then forgot - will delete